README.md

```diff
@@ -519,6 +519,11 @@ The `pointcloud_postgis` extension adds functions that allow you to use PostgreS
 > 
 >     POINT Z (-127 45 124)
 
+**PC_Envelope(pcpatch)** returns **geometry**<br/>
+**pcpatch::geometry** returns **geometry**
```

Comment:
The `geometry` function and cast were not documented before. Given that we do not plan to encourage their use I would not add them to the doc.